awx/main/analytics/subsystem_metrics.py

```diff
@@ -297,7 +290,7 @@ def pipe_execute(self):
     def send_metrics(self):
         # more than one thread could be calling this at the same time, so should
         # acquire redis lock before sending metrics
-        lock = self.conn.lock(root_key + '_lock')
+        lock = self.conn.lock(root_key + '-' + self._namespace + '_lock')
```

Comment:
So that each service (dispatcher, callback_receiver) does NOT overwrite each other